public/stylesheets/masters/reversi/reversi_site_screen_.css

```diff
@@ -329,6 +329,10 @@ fieldset fieldset.listbox {
   outline: none;
 }
 
+fieldset .footnote, .character_counter, #comment-field-description {
```

Comment:
I think the best way to fix this is actually doing to be 

```
form dd.required {
  background: #eee;
}
```

As the issue points out, the problem is only with required fields... and that's because the default stylesheet uses that selector to set the text to our default text color of #2a2a2a (otherwise, in some cases, it would show up in the red we use for required fields). 

So here we need to apply Reversi's default text color of #eee to the text of required fields.